fibheap/Documentation/attribution.txt

```diff
@@ -0,0 +1,6 @@
+All rights belong to Keith Schwarz (htiek@cs.stanford.edu) and Stanford University.
```

Comment:
@joshhayes-sheen-wf: I've written a small attribution file. I'm still not sure it's acceptable to have them here like this though. I will ask the professor.